src/ppx/graphics_util.cpp

```diff
@@ -775,6 +782,68 @@ Result CreateImageFromFile(
 
 // -------------------------------------------------------------------------------------------------
 
+Result CreateImageFromArray(
```

Comment:
Please add some comment about the function, e.g. the source data has single channel and we are writing to the R channel of an RGBA image